### PR TITLE
add http-ece

### PR DESCRIPTION
**Pull request by jsta**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jsta** [comment]
@conda-forge/help-python @conda-forge/staged-recipes ready for review

File: recipes/http-ece/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/http_ece-{{ version }}.tar.gz
+  sha256: 932ebc2fa7c216954c320a188ae9c1f04d01e67bec9cdce1bfbc912813b0b4f8
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (ocefpaf) [inline recipes/http-ece/meta.yaml:13]: This one can be noarch. Grayskull correctly detected a scripts section the metadata but it is empty. We should probably send a fix upstream.

```suggestion
  noarch: python
  script: {{ PYTHON }} -m pip install . -vv
```

File: recipes/http-ece/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  sha256: 932ebc2fa7c216954c320a188ae9c1f04d01e67bec9cdce1bfbc912813b0b4f8
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
```
Comment (ocefpaf) [inline recipes/http-ece/meta.yaml:19]:
```suggestion
    - python >=3.6
```

File: recipes/http-ece/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    - pip
+  run:
+    - python
```
Comment (ocefpaf) [inline recipes/http-ece/meta.yaml:22]: Sorry, I missed these.

```suggestion
    - python >=3.6
```

**Comment by jsta** [comment]
Thanks for the help @ocefpaf! Looks to me like its good-to-go.

**Pull request merged**